Operators/Types/lib/io/osc/OscOutput.cs

```diff
@@ -0,0 +1,142 @@
+using T3.Core.Operator;
+using T3.Core.Operator.Attributes;
+using T3.Core.Operator.Slots;
+using T3.Core.Logging;
+using System.Net;
+using Rug.Osc;
+using System;
+using T3.Core.Utils;
+
+namespace T3.Operators.Types.Id_4e99da86_482f_4037_8664_b2371526d632
+{
+    public class OscOutput : Instance<OscOutput>
+    {
+        private bool _connected;
+        private IPAddress _address;
+        private int _port;
+        private float _num_value;
```

Comment:
```suggestion
        private float _numericValue;
```
Please follow the .net c# naming conventions and avoid short forms.